### DISCUSSION TITLE
[Skia] ListViewItem interaction colors change

### DISCUSSION THREAD

**Discussion by FrozDark** [unoplatform/uno#10497] [Q&A] [answered]
How can I change a hover, pressed and other interaction color on item in a ListView since ListViewItemPresenter is not implemented on skia?
Will ContentPresenter instead working? and manually create a container for item with the interaction behaviours?

**Comment by jeromelaban** [comment, marked as answer]
At this time, you can by duplicating the style for ListViewItem, if you're using fluent styles:
https://github.com/unoplatform/uno/blob/2d870f78f45ce0eb9b7aacac5d8d772f1db95b0b/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/ListViewItem_themeresources.xaml#L232

**Reply by FrozDark** [reply]
As a lightweight workaround I've used [Platform specific XAML markup](https://platform.uno/docs/articles/platform-specific-xaml.html)
```
<!-- Redefine the brush colors by inheriting from already defined resources -->
<StaticResource x:Key="ListViewItemBackgroundPointerOver" ResourceKey="ThemedGlobalSearchHoverColor" />
<StaticResource x:Key="ListViewItemBackgroundSelected" ResourceKey="ThemedGlobalSearchSelectedColor" />
<StaticResource x:Key="ListViewItemBackgroundSelectedPointerOver" ResourceKey="ThemedGlobalSearchSelectedColor" />

<!-- Apply to non-WinUI targets using their style -->
<not_win:Style
                x:Key="ListViewItemExpanded"
                BasedOn="{StaticResource ListViewItemExpanded}"
                TargetType="ListViewItemPresenter" />
```

Then apply to a listview the item style using XAML platform specific markup
```
<ListView>
    <not_win:ListView.ItemContainerStyle>
        <Style BasedOn="{StaticResource ListViewItemExpanded}" TargetType="ListViewItem" />
    </not_win:ListView.ItemContainerStyle>
</ListView>
```
